readthedocs/proxito/README.rst

```diff
@@ -5,6 +5,41 @@ Module in charge of serving documentation pages.
 
 Read the Docs core team members can view the `Proxito design doc <https://github.com/readthedocs/el-proxito/blob/master/docs/design/architecture.rst>`_
 
+URL parts
+---------
+
+In our code we use the following terms to refer to the different parts of the URL:
+
+url:
+   The full URL, for example ``https://docs.readthedocs.io/en/latest/index.html``.
+path:
+   The whole path from the URL, for example ``/en/latest/index.html``.
```

Comment:
Does this include the GET args? What about `#fragment`

Reply:
This does not, I'll extend the definition to mention that.